doc/nrf/libraries/modem/location.rst

```diff
@@ -247,11 +247,15 @@ Configure the following options to enable location methods of your choice:
 * :kconfig:option:`CONFIG_LOCATION_METHOD_CELLULAR` - Enables cellular location method.
 * :kconfig:option:`CONFIG_LOCATION_METHOD_WIFI` - Enables Wi-Fi location method.
 
-The following options control the use of GNSS assistance data:
+For external GNSS assistance data usage, configure the following Kconfig option:
```

Comment:
```suggestion
For external GNSS assistance data usage, set the following Kconfig option:
```

Reply:
As there is only one bullet below, you could also formulate this like:

For...., set the :kconfig:option:`CONFIG_LOCATION_SERVICE_EXTERNAL` Kconfig option.
It enables...

And so on.